assets/global.js

```diff
@@ -1247,18 +1257,11 @@ class BulkAdd extends HTMLElement {
     } else {
       event.target.setCustomValidity('');
       event.target.reportValidity();
+      event.target.setAttribute('value', inputValue);
       this.startQueue(index, inputValue);
     }
   }
 
-  getSectionsUrl() {
-    if (window.pageNumber) {
-      return `${window.location.pathname}?page=${window.pageNumber}`;
-    } else {
-      return `${window.location.pathname}`;
-    }
-  }
```

Comment:
This was only used by the QuickOrderBulk subclass, no need for it to be on the parent.